lib/wanda/policy.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Wanda.Policy do
+  @moduledoc """
+  Handles integration events.
+  """
+
+  alias Cloudevents.Format.V_1_0.Event, as: CloudEvent
+  alias Wanda.Execution.Fact
+
+  @facts_gathered_event "trento.checks.v1.FactsGathered"
+
+  @spec handle_event(CloudEvent.t()) :: :ok | {:error, any}
```

Comment:
Is this signature correct?
Shouldn't return `{:ok, [Fact.t()]}` or similar?

Reply:
nope it returns :ok because it sends a message to the genserver